lib/models/models.dart

```diff
@@ -136,6 +138,45 @@ class BalanceModel {
   int get hashCode => super.hashCode;
 }
 
+class RateModel {
+  final double numericQuantity;
+
+  RateModel(
+    this.numericQuantity
+  );
+
+  factory RateModel.fromJson(Map<String, dynamic> json) {
+    if (json != null && json.isNotEmpty) {
+      return RateModel(_parseQuantityString(json["rows"][0]["current_seeds_per_usd"] as String));
+    } else {
+      return RateModel(0);
+    }
+  }
+
+  static double _parseQuantityString(String quantityString) {
+    if(quantityString == null) {
+      return 0;
+    }
+    return double.parse(quantityString.split(" ")[0]);
+  }
+
+  double convert(double seedsAmount) {
+    return numericQuantity > 0 ? seedsAmount / numericQuantity : 0;
```

Comment:
I would have thought that the rate tells us how much 1 SEED is worth in USD, but in that case the calculation would have been seedsAmount * numericQuantity. So how does it work? What is the rate number we get from on-chain?